src/mock/points.js

```diff
@@ -0,0 +1,117 @@
+import {getRandomArrayElement, getRandomInt} from '../utils';
+import dayjs from 'dayjs';
+
+const POINT_TYPES = ["taxi", "bus", "train", "ship", "drive", "flight", "check-in", "sightseeing", "restaurant"];
+const DESTINATIONS_DESCRIPTIONS = [
+  'Aliquam id orci ut lectus varius viverra',
+  'tortor ac porta dapibus',
+  'non porta ligula feugiat eget'
+];
+const CITY_NAMES = [
+  'Chamonix',
+  'Chicago',
+  'London',
+  'Moscow',
+  'Paris'
+];
+const PICTURES_DESCRIPTIONS = [
+  'ipsum dolor sit amet',
+  'purus sit amet tempus',
+  'condimentum sed nibh vitae'
+];
+const OFFERS_TITLES = [
+  'Order Uber',
+  'Add luggage',
+  'Switch to comfort',
+  'Rent a car',
+  'Add breakfast',
+  'Book tickets'
+];
+
+export function generateDestinations(count) {
+  let data = [];
+  for (let i = 0; i < count; i++) {
+    data.push(
+      {
+        "id": i+1,
```

Comment:
будет то же самоно но по мне красивей код
```suggestion
  for (let i = 1; i <= count; i++) {
    data.push(
      {
        "id": i,
```